app/services/markdown_processor/parser.rb

```diff
@@ -89,7 +89,7 @@ def evaluate_listings_markdown
       markdown = Redcarpet::Markdown.new(renderer, Constants::Redcarpet::CONFIG)
       allowed_tags = %w[strong abbr aside em p h4 h5 h6 i u b code pre
                         br ul ol li small sup sub a span hr blockquote kbd]
-      allowed_attributes = %w[href strong em ref rel src title alt class]
+      allowed_attributes = %w[href strong em ref rel src title alt]
```

Comment:
Not necessarily as part of this PR, but given this repetition, should we extract these somewhere?